### ISSUE TITLE
Running status is not rendered correctly.

### ISSUE THREAD

**Issue by rahulr56** [jenkinsci/embeddable-build-status-plugin#38]
Running status is not rendered correctly.

![image](https://user-images.githubusercontent.com/8971145/53767381-7fad8d00-3e8a-11e9-804d-5f63f84ac2e2.png)

**Comment by thomas-dee** [comment]
Well, this is actually intended behavior.  "running" means:
1. It is a running process -> the status is animated with a pulsing color
1. The result is not clear yet -> the color is blue (and not red, green or yellow which would indicate a final result)
1. The result of the previous run is indicated by the colored bar on the right side. In your example: green.

You can change the animated color (blue) using the query parameter [`animatedOverlayColor`](https://github.com/jenkinsci/embeddable-build-status-plugin#color-and-animatedoverlaycolor).

Examples: `...?animatedOverlayColor=brightgreen` or `...?animatedOverlayColor=red`

**Comment by rahulr56** [comment]
@thomas-dee Yes, I do get the states. The annoying part is the blue and green overlap in the image (highlighted in red)

![image](https://user-images.githubusercontent.com/8971145/53825707-c0a9ae00-3f2b-11e9-9c4e-16cab39a1a63.png)

**Comment by thomas-dee** [comment]
@rahulr56 This overlap is also intended. As stated in my previous post, those overlaps indicate the previous run result.
If you find this "annoying" I might add another query parameter allowing to disable that.

**Comment by rahulr56** [comment]
Yup, that would be great. Thanks 😊

**Comment by Makr91** [comment]
Maybe put it on the left side instead?

**Comment by hogarthj** [comment]
An oddity I'm seeing here is that if I define the animatedOverlayColor on the query the image doesn't "settle" like a normal progress -> finished state ... instead it keeps on pulsing even when the job is complete.

Again, this is only if animatedOverlayColor is defined on the request.
